extensions/docker/src/extension.ts

```diff
@@ -168,14 +167,7 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   }
 
   // monitor daemon
-  monitorDaemon(extensionContext).catch((err: unknown) => {
-    console.error('Error while monitoring docker daemon', err);
-    if (err instanceof Error) {
-      extensionApi.env.createTelemetryLogger().logError(err);
-    } else {
-      extensionApi.env.createTelemetryLogger().logError(err.toString());
-    }
-  });
+  void monitorDaemon(extensionContext);
```

Comment:
hello, here it looks you're bypassing the linter and do not catch error on promises, etc.